### PR TITLE
Laravel 8 support

### PR DESCRIPTION
**Pull request by marvinhosea**
#### Changes made
- Added support for laravel ^8.0
- Added support for illuminate/support ^8.0
- Added support for illuminate/notifications ^8.0
- Added support for guzzlehttp/guzzle ^7.0

Changed AfricasTalkingLtd/africastalking-php package to appslabke/africastalkin, since Africas Talking is delaying PRs merge

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -10,22 +10,21 @@
...
+        "email": "osaigbovoemmanuel1@gmail.com",
+        "role": "Developer"
+    }],
     "require": {
         "php": "^7.2.5",
-        "africastalking/africastalking": "^2.3",
-        "illuminate/notifications": "~5.5 || ~6.0 || ~7.0",
-        "illuminate/support": "~5.5 || ~6.0 || ~7.0"
+        "appslabke/africastalking": "^2.5.1",
+        "guzzlehttp/guzzle": "^7.0.1",
```
Comment (atymic) [inline composer.json]: Can we also allow guzzle 6?

File: composer.json [hunk 2]
```diff
@@ -10,22 +10,21 @@
...
+        "illuminate/notifications": "~5.5 || ~6.0 || ~7.0 || ^8.0",
```
Comment (atymic) [inline composer.json:22]: Can drop 5.x now

Comment (atymic) [reply composer.json:22]: (and same below)

**Comment by atymic** [comment]
Ideally i'd prefer not to have to fork it, but it does look like it's been a while.
Will give them a few more days and if not we can merge it.

File: composer.json (hunk 2 above)
Comment (marvinhosea) [reply composer.json:22]: I am sure some people are still using 5.x

File: composer.json (hunk 1 above)
Comment (marvinhosea) [reply composer.json]: Sure lemme include it too

Comment (marvinhosea) [reply composer.json]: Fixed with this commit 44557fb717b282659250de66bdd0ab3274fbdc13

File: composer.json (hunk 2 above)
Comment (atymic) [reply composer.json:22]: It's out of support (both security and bug fix) but ok to leave it.

**Pull request merged**